ppr-ui/.eslintrc.js

```diff
@@ -7,8 +7,13 @@ module.exports = {
     'plugin:vue/essential',
     '@vue/standard',
     '@vue/typescript'
+    // ToDo: enable and fix Lint issues when capacity.
+    // 'plugin:vue/base',
+    // 'plugin:vuetify/base',
+    // 'plugin:vuetify/recommended'
```

Comment:
With the plugin available, we can enable these and start to implement vuetify fixes as we go. 
This will help alleviate the deprecation fixes required when we upgrade vuetify.

Reply:
Enabled these, after fixing issues 👍